classes/ThankYous/ThankYou.php

```diff
@@ -0,0 +1,227 @@
+<?php
+
+namespace Claromentis\ThankYou\ThankYous;
+
+use Claromentis\ThankYou\Comments\CommentableThankYou;
+use Claromentis\ThankYou\Tags\Tag;
+use Claromentis\ThankYou\Thankable\Thankable;
+use Date;
+use InvalidArgumentException;
+use User;
+
+class ThankYou
+{
+	/**
+	 * @var User $author
```

Comment:
Same as the `Tag` class, the variable names aren't necessary in the `@var` annotations of class properties.

Reply:
Fixed in 396cdf3be76461324e5cbc0f6f812b0af263f5be